faq/index.md

```diff
@@ -2,153 +2,280 @@
+Service disruptions are possible for both commercial ISPs and NYC Mesh. Extreme weather such as heavy rain, blizzards and hail, for example, can decrease connection speeds as they affect the router’s wireless signal. Our community actively monitors the health of the network, and, unlike commercial ISPs, responds quickly to service requests.
 
-It is possible to connect directly to other people on the mesh without using the internet. If for some reason the internet is shut down by a fascist dictator, the mesh will still work. Internet shutdowns are becoming increasingly common in the world. 
+## <a name=""></a>Is the mesh secure?
 
-The mesh has small web servers that show local information and allow you to chat and post messages. So far we have servers in five locations- Harlem, Bay Ridge, 3rd St. and 7th St. in the East Village, and our main directory "wiki.mesh" in Williamsburg. We are in need of volunteers to help add content and maintain our servers.
+By default, our mesh router is firewalled from your local network. It is not possible to reach beyond the mesh router to your local access network (LAN). 
 
-## <a name="why-join"></a>Why should I join the mesh now?
+Network traffic within the mesh is not encrypted by default. When connecting through the mesh you should use the same standard precautions as you would when connecting to Wi-Fi at a coffee shop or airport: use [https](https://en.wikipedia.org/wiki/HTTPS) (lock icon) web sites for secure connections (most browsers do this by default nowadays),  or use a VPN service.
 
-By joining the mesh you help us grow and increase our reach across the city. This is a community project and you will be helping to create a decentralized infrastructure, free from traditional business interests.
+If you connect to the mesh network indirectly over the internet (for example over a cable ISP connection) this is by [tinc VPN](https://www.tinc-vpn.org/) and is secure.
 
-You can also help your neighborhood by providing access points for local parks, cafes and bars. Local businesses are often keen to host a node once you explain the benefits to them.
 
-## <a name="coverage"></a>Is it possible for the mesh to cover the whole country?
+## <a name="userdata"></a>What is your user data policy?
 
-Yes, there is a mesh network called "Guifi" in Spain that covers a large part of the country. In order to do this, they have a backbone of many long distance WiFi connections and also community owned fiber.
+NYC Mesh does not collect, store, or log any user traffic or content that passes through our network.
 
-## <a name="isp"></a>Are you an Internet Service Provider (ISP)?
+NYC Mesh may periodically look at [traffic headers](https://en.wikipedia.org/wiki/IPv6_packet) for troubleshooting purposes as traffic passes through the NYC Mesh network, however, this data is never logged or stored. NYC Mesh may collect aggregate statistics to monitor traffic flow to ensure a successful network.
```

Comment:
Header link would be better if it pointed to https://en.wikipedia.org/wiki/IPv4#Header

Reply:
sure, will do.